packages/system/src/Item/ListView/api.ts

```diff
@@ -1,57 +0,0 @@
-import {
```

Comment:
we can't do anything other than read system data, so using this interface seemed a bit silly?